myth

```diff
@@ -330,66 +302,51 @@ if args.storage:
     if not args.address:
         exitWithError(args.outform, "To read storage, provide the address of a deployed contract with the -a option.")
     else:
-        position = 0
-        length = 1
-        array = 0
-
+        (position, length) = (0, 1)
         try:
-            params = (args.storage).split(",")
-            if len(params) >= 1 and len(params) <= 3:
-                position = int(params[0])
-            if len(params) >= 2 and len(params) <= 3:
-                length = int(params[1])
-            if len(params) == 3:
-                if re.match("array", params[2]):
-                    array = 1
+            params = args.storage.split(",")
             if len(params) >= 4:
                 exitWithError(args.outform, "Invalid number of parameters.")
+
+            if len(params) >= 1:
+                position = int(params[0])
+            if len(params) >= 2:
+                length = int(params[1])
+            if len(params) == 3 and params[2] == "array":
+                position_formatted = str(position).zfill(64)
+                position = int(utils.sha3(position_formatted), 16)
+
         except ValueError:
             exitWithError(args.outform, "Invalid storage index. Please provide a numeric value.")
 
-        if array:
-            position_formated = str(position).zfill(64)
-            position = int(utils.sha3(position_formated), 16)
-
         try:
             if length == 1:
-                print("{}: ".format(position) + eth.eth_getStorageAt(args.address, position))
+                print("{}: {}".format(position, eth.eth_getStorageAt(args.address, position)))
             else:
                 for i in range(position, position + length):
-                    print("{}: ".format(hex(i)) + eth.eth_getStorageAt(args.address, i))
+                    print("{}: {}".format(hex(i), eth.eth_getStorageAt(args.address, i)))
```

Comment:
A question: why we need `hex()` to the `i`(which may be a `position`), but we don't need it in previous `print("{}: ".format(position)`?